content/livestream.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Live Stream"
+menu: "main"
+weight: 2
+---
+
+## Watch the conference online
+
+Join us virtually in two live streams on April 4th and April 5th via the [DataLad YouTube Channel](https://www.youtube.com/@DataLad/streams)!
+
+* Live stream of conference day 1: [Take me there](https://youtube.com/live/BwRy3z_hQ70?feature=share)
+* Live stream of conference day 2: [Take me there](https://youtube.com/live/L1MKaUgg1xs?feature=share)
+
+During and after a talk, you can ask questions via the [Distribits meeting Q&A](https://matrix.to/#/!zgDTuNhNsLsAWNOtcs:matrix.org?via=matrix.org&via=matrix.uni-marburg.de&via=inm7.de) Matrix Room.
```

Comment:
```suggestion
During and after a talk, you can ask questions via the **Distribits meeting Q&A** Matrix Room. The link to this room has been sent to all attendees via email
```

To remain consistent with the virtual participation being under the CoC and the youtube comments being disabled, should we not remove the link to the matrix channel from the website?

Reply:
that would also work for me :+1:, thanks for the suggestion. Given that not too many people have joined the space, though, I wonder if we should send a reminder about this so that people have it at the top of their inbox. Alternatively, there is a link in the logistics slides (but we would need to host the slides somewhere to make it actually useful)

Reply:
I think another reminder email makes sense yes.